### PR TITLE
🌐 Add Italian translation for `docs/it/docs/index.md`

### PR DESCRIPTION
**Pull request by matteospanio**
add italian translation for index docs page based on PR  #1576 by @csr  and reviewed by @bsab, @PieroMacaluso and @rpalloni

### CODE REVIEW

File: docs/it/docs/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
 </p>
 <p align="center">
-    <em>FastAPI framework, high performance, easy to learn, fast to code, ready for production</em>
+    <em>FastAPI framework, alte prestazioni, facile da imparare, rapido da programmare, pronto per il rilascio in produzione</em>
```
Comment (Ale-Cas) [inline docs/it/docs/index.md:9]: Minor: `rapido da implementare` sounds a bit better than `rapido da programmare`

File: docs/it/docs/index.md [hunk 2]
```diff
@@ -290,174 +289,176 @@ def update_item(item_id: int, item: Item):
...
-* Validation of data:
-    * Automatic and clear errors when the data is invalid.
-    * Validation even for deeply nested JSON objects.
-* <abbr title="also known as: serialization, parsing, marshalling">Conversion</abbr> of input data: coming from the network to Python data and types. Reading from:
+* Supporto per gli editor di testo, incluso:
+    * Autocompletamento.
+    * Controllo sulle annotazioni di tipo.
+* Validazione dei dati:
+    * Errori chiari e automatici quando i dati sono invalidi.
+    * Validazione anche per gli oggetti JSON altamente nidificati.
```
Comment (Ale-Cas) [inline docs/it/docs/index.md:347]: I would change `altamente nidificati` with `più complessi`

File: docs/it/docs/index.md [hunk 3]
```diff
@@ -290,174 +289,176 @@ def update_item(item_id: int, item: Item):
...
+* Validerà che `item_id` sia di tipo `int` per le richieste `GET` e `PUT`.
+    * Se non lo è, il client vedrà un errore chiaro e utile.
+* Controllerà se ci sia un parametro opzionale chiamato `q` (per esempio `http://127.0.0.1:8000/items/foo?q=somequery`) per le richieste `GET`.
+    * Siccome il parametro `q` è dichiarato con `= None`, è opzionale.
+    * Senza il `None` sarebbe stato obbligatorio (come per il body della richiesta `PUT`).
+* Per le richieste `PUT` su `/items/{item_id}`, leggerà il body come JSON, questo comprende:
+    * verifica che la richiesta abbia un attributo obbligatorio `name` e che sia di tipo `str`.
+    * verifica che la richiesta abbia un attributo obbligatorio `price` e che sia di tipo `float`.
+    * verifica che la richiesta abbia un attributo opzionale `is_offer` e che sia di tipo `bool`, se presente.
+    * Tutto questo funzionerebbe anche con oggetti JSON altamente nidificati.
```
Comment (Ale-Cas) [inline docs/it/docs/index.md:380]: same as above

**Review by Ale-Cas — APPROVED** [review]

**Comment by matteospanio** [comment]
Since it was a one year old PR I updated the translation with @Ale-Cas contributions and I pulled the latest code update  from master branch.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 96b0efbba3750ef8c77df0a6eb82b791a5ce1cc6 at: https://100687c6.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit a19275d1157a0dc3a184829df146011d1d9145ef at: https://814637cf.fastapitiangolo.pages.dev

**Pull request merged**